integration_tests/Dockerfile-agentd

```diff
@@ -5,4 +5,4 @@ ENV PYTHONDONTWRITEBYTECODE='true'
 COPY . /usr/src/wazo-agentd
 
 WORKDIR /usr/src/wazo-agentd
-RUN python setup.py develop
+RUN python3 -m pip install -e .
```

Comment:
Any reason/advantage to use python module instead of the cli directly?

Reply:
The reason is conservatism, using the same recommended approach for a normal debian environment and be as explicit as possible. But indeed it is not necessary in docker where we assume there is only one python and one pip.